Moosh/Command/Moodle36/Category/CategoryResortCourses.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * moosh - Moodle Shell
+ *
+ * @copyright  2012 onwards Tomasz Muras
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+namespace Moosh\Command\Moodle36\Category;
+
+use Moosh\MooshCommand;
+use core_course_category;
+
+class CategoryResortCourses extends MooshCommand
+{
+    public function __construct()
+    {
+        parent::__construct('resortcourses', 'category');
+
+        $this->addArgument('category_id');
+        $this->addArgument('sort');
+
+        $this->maxArguments = 2;
+    }
+
+    public function execute()
+    {
+        global $DB;
+
+        list($categoryid, $sort) = $this->arguments;
+        if (!$cattosort = $DB->get_record('course_categories', array('id'=>$categoryid))) {
+            cli_error("No category with id '$categoryid' found");
+        }
+        else {
+            $this->resortcourses_category($cattosort, $sort);
+        }
+    }
+
+    protected function resortcourses_category($category, $sort)
+    {
+        global $CFG;
```

Comment:
Great contribution.
I would advise removing this unused global variable.

Reply:
You are right! I'm even unsure about the correct location of the file. What is the best way to determine the correct moodle version? Feel free to move.
Thank you!

Reply:
moosh will check the current Moodle version and will run the appropriate version of the command. That means, from the best matching namespace.

Since this command has been put under Moosh/Command/Moodle36, then moosh will run it for Moodle 36.
Since there is no CategoryResortCourses.php under Moodle37, Moodle38, Moodle39 - then moosh will run exactly the same script against later Moodle version as well (hoping for the best!).